packages/blitz-rpc/src/loader-client.ts

```diff
@@ -39,12 +41,24 @@ export async function transformBlitzRpcResolverClient(
 ) {
   assertPosixPath(id)
   assertPosixPath(root)
-
   const resolverFilePath = "/" + posix.relative(root, id)
   assertPosixPath(resolverFilePath)
   const routePath = convertPageFilePathToRoutePath(resolverFilePath, options?.resolverPath)
   const resolverName = convertFilePathToResolverName(resolverFilePath)
   const resolverType = convertFilePathToResolverType(resolverFilePath)
+  const resolverConfig: ResolverConfig = {
+    httpMethod: "POST",
+  }
+  if (resolverType === "query") {
+    try {
+      const {httpMethod} = getResolverConfig(id)
```

Comment:
you already have `src` as the first argument to this `transformBlitzRpcResolverClient` function, so you can directly call `parseResolverCacheMiss(src)` instead of reading the file again from disk.